src/StaticFAQ/ArticleFeedback/ScreenInput.js

```diff
@@ -0,0 +1,107 @@
+// @flow
+
+import * as React from 'react';
+import css from 'styled-jsx/css';
+import { Close } from '@kiwicom/orbit-components/lib/icons';
+
+import { Box, ContactPageLink } from '../../common';
+import createComment from '../../mutations/CreateCommentMutation';
+import screensList from './screensList';
+
+type Props = {|
+  changeScreen: (nextScreen: string) => void,
+  articleId: string,
+|};
+type State = {|
+  comment: string,
+|};
+
+const style = css`
+  div.title {
```

Comment:
I think this title should be **bold**

Reply:
Disagree. As per design `font-weigth: 500`

![image](https://user-images.githubusercontent.com/2466887/39757284-aba2df32-52cc-11e8-96a9-697f8453688c.png)